Tests/Domain/SecurityIdentityRetrievalStrategyTest.php

```diff
@@ -284,7 +284,7 @@ public function getRoles(): array
         return [];
     }
 
-    public function eraseCredentials()
+    public function eraseCredentials(): void
```

Comment:
befire this PR, we should have one on 3.x that adds all missing `@return` don't you think?

Reply:
Yes.